app/views/users/edit.html.haml

```diff
@@ -3,7 +3,7 @@
 = form_tag edit_path, :class => '' do
 
   = label_tag(:age, 'Age')
-  = number_field_tag :age, 'age', class: ""
+  = number_field_tag :age, 'age', min: 1, required: true
```

Comment:
perhaps we should keep the class attribute for later CSS/bootstrap styling on the form

Reply:
resolved, I added it back in